test/get-set-delete.test.ts

```diff
@@ -1,221 +1,232 @@
 import {SetupIntegrationTest, isRedisBackedTest} from './integration-setup';
```

Comment:
if you are reviewing this you will want to toggle "hide whitespace changes" on github.